command/debug.go

```diff
@@ -481,7 +482,19 @@ func (c *DebugCommand) collectConsul(dir, consul string) error {
 
 	token := c.consulToken
 	if token == "" {
-		os.Getenv("CONSUL_TOKEN")
+		token = os.Getenv("CONSUL_HTTP_TOKEN")
+	}
+	if token == "" {
+		file := os.Getenv("CONSUL_HTTP_TOKEN_FILE")
+		if file != "" {
+			fh, err := os.Open(file)
+			if err == nil {
+				bs, err := ioutil.ReadAll(fh)
+				if err == nil {
+					token = strings.TrimSpace(string(bs))
+				}
+			}
+		}
```

Comment:
We don't support this elsewhere, so it's not necessary here.

That being said it seems like a Great Idea, so do you mind just opening a new issue for implementing this holistically (and hopefully sharing code)?